### ISSUE TITLE
Use tags jump to definition too slow（almost 3 seconds and more）with ctrl + ]

### ISSUE THREAD

**Issue by Cloudprogram** [vim/vim#4575]
_Instructions: Replace the template text and remove irrelevant text (including this line)_

**Describe the bug**
(Issues related to the runtime files should be reported to their maintainer, check the file header.)

**To Reproduce**
Detailed steps to reproduce the behavior:
1. use the popup menu(Open With Vim Tab or edit with vim（用vim编辑）) for the right mouse button to open start.S(u-boot-1.1.6\cpu\arm920t)
2. :set tags =E:\program\u-boot-1.1.6\tags
3. ctrl + ] in function start_armboot to jump to definition in line 333（）
4. after 3s it would jump to definition，it takes too long
if run gvim.exe with left mouse button ，then drag the  start.S to gvim，then do process 2，3
it would jump to definition very fast。
what‘s the difference between right mouse button and left mouse button？？？

**Expected behavior**
very fast to jump to definition
**Screenshots**
If applicable, copy/paste the text or add screenshots to help explain your problem.
![123](https://user-images.githubusercontent.com/49339934/59971126-c691d080-95a8-11e9-9028-f84f2f779ed2.jpg)
![234](https://user-images.githubusercontent.com/49339934/59988737-c6f49f00-966e-11e9-8b27-f5b32c1e91ae.png)
![7](https://user-images.githubusercontent.com/49339934/59988741-c956f900-966e-11e9-8fcd-e698856a9868.png)

![15](https://user-images.githubusercontent.com/49339934/59971136-ec1eda00-95a8-11e9-9c75-bfa7d1b6b3b7.jpg)
![46](https://user-images.githubusercontent.com/49339934/59971235-b5e25a00-95aa-11e9-88a0-a3db3c5a4e64.jpg)
![4757](https://user-images.githubusercontent.com/49339934/59971772-2db58200-95b5-11e9-96c9-ce42620105f1.jpg)
![35](https://user-images.githubusercontent.com/49339934/59971929-b33a3180-95b7-11e9-8775-cbc9351ec6ce.jpg)
![457](https://user-images.githubusercontent.com/49339934/59971931-b503f500-95b7-11e9-9ec1-f7c28706bd60.jpg)

**Environment (please complete the following information):**
 - Vim version [8.1] 
 latest version gvim_8.1.1573_x64_signed.zip in https://github.com/vim/vim-win32-installer/releases
 - OS: [, Windows 7 ]
 - Terminal: [gvim] 

**Additional context**
tags （u-boot-1.1.6\tags）
If i run gvim with double click or its shortcuts in desktop（gVim 8.1） ，it work normally。
why it takes so long to jump to definition with the popup menu ？？？
project u-boot-1.1.6 sotre in a Hard Disk Drive(HDD)
vim was install in a Solid State Disk(SDD),which is faster than HDD.
copy project u-boot-1.1.6 to SDD from HDD,jumping to definition  is also fast with the same behavior（gvim start with the popup menu “Open With Vim Tab”）。

/**/
In conclusion.When path is in network or low speed HDD,vim would work slowly. 
[u-boot-1.1.6.zip](https://github.com/vim/vim/files/3317452/u-boot-1.1.6.zip)
